ci-operator/step-registry/hypershift/install/hypershift-install-ref.yaml

```diff
@@ -13,6 +12,10 @@ ref:
   - mount_path: /etc/hypershift-pool-aws-credentials
     name: hypershift-pool-aws-credentials
     namespace: test-credentials
+  from_image:
+    namespace: ocp
+    name: "4.13"
+    tag: hypershift-operator
```

Comment:
Reviewer Note:  would like to use `from: hypershift-operator` here, but this is used on the `hypershift-aws-conformance` flow that is called from many repos whose configs do not set `hypershift-operator` in their `base_images`.

Reply:
wouldn't this change break all our CI pipelines which are all using this ref?

This will always set the hypershift CLI used to install the HO to 4.13, but the HO image `HYPERSHIFT_RELEASE_LATEST` installed will have a different version than the CLI according to the branch/pipeline defined `hypershift-operator` image.

Reply:
Back when I did this PR, the assumption was that only HO from main (4.13 at the time) was supported.   We've since changed our position on this supporting the HO in both `main` and `release-4.13`.  If we accept the `hypershift-operator` ref from the top level job, that will sometimes be from 4.12 or, in this case, even 4.11, which we definitely don't backport HO changes that far.